assets/js/media-upload.js

```diff
@@ -88,5 +109,5 @@ jQuery(document).ready(function($) {
 		$preview.remove();
 		$( this ).remove();
 		$( '.attachment-resolution, .attachment-resolution-warning' ).remove();
```

Comment:
I think this should also be updated. When I remove the image for one language, the notice will be removed in all tabs.

Reply:
@MohamadNateqi Nice catch, well done! I have updated the code.